### ISSUE TITLE
Add an alternate view (list with large icons and Entry-Preview-Panel on the right)

### ISSUE THREAD

**Issue by OLLI-S** [keepassxreboot/keepassxc#4244]
## Summary
You should offer an **optional** new layout of the KeePassXC window that looks like modern password managers.

## Details
When I look at other password managers like Enpass or 1Password, they offer a window layout like this:

![image](https://user-images.githubusercontent.com/8037790/73137843-c4bb4e80-405c-11ea-9eec-c5647bb13b1b.png)
(screen shots taken from https://www.enpass.io/)

![image](https://user-images.githubusercontent.com/8037790/73137875-0815bd00-405d-11ea-86ad-e825c1be8655.png)
(screen shots taken from https://1password.com/tour/)

The window has the following elements:
- On the left you have a tree view like you have it in KeePassXC, so this is perfect
- Below the tree there is in Enpass the "Password Audit" section, where you can see the list of pwned password, weak passwords or identical passwords.
If you integrate this new layout you may add your Password Health Check (#3993) here.
- Then they show a list of password entries of the selected folder/category (like you have).
In difference of KeePassXC they show a larger icon, left to the icon they show the title of the element in bold font and below the title they show additional information, like the username or the card number (of credit cards).
- On the right they show the Entry-Preview-Panel (that you already have).
Here they display a larger icon (referring to the screenshots of the website), the title in bold font and all fields of the entry below each other (a list of fields).
Custom fields are also shown in this list and users can change the order of the fields by drag & drop (I tested this in Enpass).

This would be a total new layout.
Maybe you start with offering a new view where you display large icons (32 x 32 pixel) in the list of entries.

**Imortant**
This layout is optional and users can select in the settings if they like the classic layout or the modern layout.
By default they see the classic layout.

**Comment by ArminWeb** [comment]
Such a modern UI is the main reason why people are using other password managers (I talked to colleagues and they told me that KeePass/KeePassXC is ugly and they prefer 1Password).
IMHO this new layout should be the default layout!

**Comment by phoerious** [comment]
We're in the process of revamping the UI, although I am not sure if this three column layout is ideal. It looks fancy, but it's also quite cluttered and impractical if you are managing a larger number of passwords. It's hard to focus on something when you have three visually almost identically important columns and you have very few passwords in view at once.

**Comment by droidmonkey** [comment]
I much prefer our current layout. The preview panel can be revamped to emphasize various attributes and be more "modern".

**Comment by OLLI-S** [comment]
@droidmonkey to work over the preview panel is a good idea.
@droidmonkey and @phoerious what do you say about my suggestion to select between "Small Icons" (16 x 16 pixels) and "Large Icons" (32 x 32 pixels) in the list of password entries?

**Comment by droidmonkey** [comment]
Please don't make a new issue this one is more than sufficient. We have too many issues.

**Comment by OLLI-S** [comment]
OK, I will not create new issues.

So this issue addresses several suggestions:

1. add an option to show the preview panel right to the password list
2. select to see "Small Icons" (16 x 16 pixels) and "Large Icons" (32 x 32 pixels) in the list of password entries
3. revamp the preview panel to emphasize various attributes and be more "modern"

Glad, that this is on the wish list (and not closed).

**Comment by MarkWieczorek** [comment]
As someone who uses both KeyPassXC and 1Password, I would have to agree that a simplified user interface would be highly desirable, both for experts and newcomers alike.

It seems to me that the first step in achieving this goal would be to offer the user the option to have either a vertical layout (like it is now), or a horizontal layout (like most other password managers, as shown in the screenshot above). 

In addition to the other points discussed above, a simplified interface should probably combine the "general", "advanced", and "autotype" tabs into a single windown.

**Comment by NomarCub** [comment]
> a simplified interface should probably combine the "general", "advanced", and "autotype" tabs into a single windown.

Is there a separate issue for this?
This is my main pain point, and what makes me want to go back to [KeeWeb ](https://github.com/keeweb/keeweb) even after months. Having to click around to see all data for a single entry is infuriating.

> We are a different application with a different audience than those other apps. That must be taken into consideration.

KeePassXC could have the audience of "those other apps" too, judging by it's statement in the readme. There's no real modern alternative on desktop as far as I looked, and you're leaving people behind if you stick to old UI conventions.

**Comment by Dominic-t3ch** [comment]
I created a new issue with this suggestion (option to move the preview panel to the right). Unfortunately it has been closed with reason that it is not planned (and duplicate).

February 2025, I still think that it's a great idea to have right preview panel like other password managers.

**Comment by DerDemystifier** [comment]
I second this. With ultra-wide screens now widely available, making use of abundant width instead of limited height is now almost a rule of thumb. I don't know who would prefer the preview panel to be at the bottom.

**Comment by Dominic-t3ch** [comment]
> I second this. With ultra-wide screens now widely available, making use of abundant width instead of limited height is now almost a rule of thumb. I don't know who would prefer the preview panel to be at the bottom.

You're right. Since many years we have wide screens. Most softwares and services now use vertical aligned panels. I can't remember the last time I used a preview panel at bottom (like in emails). The only one is KeePassXC.
